ipaserver/install/cainstance.py

```diff
@@ -278,10 +278,12 @@ class CAInstance(DogtagInstance):
        2 = have signed cert, continue installation
     """
 
-    tracking_reqs = ('auditSigningCert cert-pki-ca',
-                     'ocspSigningCert cert-pki-ca',
-                     'subsystemCert cert-pki-ca',
-                     'caSigningCert cert-pki-ca')
+    tracking_reqs = {
```

Comment:
Would it be reasonable to add a comment line a description of the key/value pairs being the nickname and profile name?